examples/app/stock_applet/stock_app.py

```diff
@@ -198,10 +217,13 @@ def input_change(self, obj, attrname, old, new):
             self.ticker2 = new
         if obj == self.ticker1_select:
             self.ticker1 = new
-        self.make_source()
-        self.make_plots()
-        self.set_children()
-        curdoc().add(self)
+
+        # avoid trying to compare same stocks
+        if self.ticker2 != self.ticker1:
```

Comment:
Not sure about this... when I try it and I did not see any update or any message about I am trying to compare the same stock... seems that the app is not working... I guess I prefer the comparative between the same stock, even if it is redundant... other @bokeh/core thoughs?